example/macos/Runner/AppDelegate.swift

```diff
@@ -1,9 +1,13 @@
 import Cocoa
 import FlutterMacOS
 
-@NSApplicationMain
+@main
 class AppDelegate: FlutterAppDelegate {
   override func applicationShouldTerminateAfterLastWindowClosed(_ sender: NSApplication) -> Bool {
     return true
   }
+
+  override func applicationSupportsSecureRestorableState(_ app: NSApplication) -> Bool {
+    return true
+  }
```

Comment:
Also automated when running using Flutter `main` channel (not specific to SPM):

```console
macos/Runner/AppDelegate.swift does not override
applicationSupportsSecureRestorableState. Updating.
```

Reply:
@EchoEllet Can you add this changes for ios as well?

Reply:
This is macOS specific, running `flutter build ios --config-only` wouldn't update the example iOS app.